channeldb/graph.go

```diff
@@ -2107,6 +2112,16 @@ func (c *ChannelGraph) FilterKnownChanIDs(chansInfo []ChannelUpdateInfo,
 
 	var newChanIDs []uint64
 
+	// isStale returns whether the timestamp is too far into the past.
+	isStale := func(timestamp time.Time) bool {
+		return time.Since(timestamp) > ChannelUpdateExpiryPeriod
+	}
+
+	// isSkewed returns whether the timestamp is too far into the future.
+	isSkewed := func(timestamp time.Time) bool {
```

Comment:
Maybe some services rely on setting the chanupdate far in the future to not have to update gossip very often ?

Reply:
perhaps but I think they should not be allowed to do this (according to the spec too iirc) & so we should not need to accommodate  for that behaviour